send_test.go

```diff
@@ -0,0 +1,75 @@
+package courier_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/trycourier/courier-go"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSend(t *testing.T) {
+
+	type Data struct {
+		Foo string
+	}
+	type Profile struct {
+		Email string
+	}
+	type RequestBody struct {
+		Event     string
+		Recipient string
+		Profile   Profile
+		Data      Data
+	}
+
+	var requestBody RequestBody
+
+	expectedResponseID := "123456789"
+	server := httptest.NewServer(http.HandlerFunc(
+
+		func(rw http.ResponseWriter, req *http.Request) {
+
+			assert.Equal(t, "/send", req.URL.String())
+
+			decoder := json.NewDecoder(req.Body)
+			err := decoder.Decode(&requestBody)
+			if err != nil {
+				t.Error(err)
+			}
+
+			rw.Header().Add("Content-Type", "application/json")
+			// nolint
```

Comment:
@yomashExpel out of curiosity, why was lint skipped here?

Reply:
it was for the ignored returned values, as I didn't need it for the purposes of the test I was writing

Reply:
actually, i went back to double check and its for ignoring the return error, which I shouldn't.  I will remove and handle that error